src/utils/normalize-mime-parts.ts

```diff
@@ -146,6 +148,9 @@ export function normalizeMimeParts(
 					}
 
 					isBody = true;
+				} else if (!bodyType && disposition === 'inline' && MORE_TEXT_TYPES.includes(type)) {
```

Comment:
too many if/else conditions can we simplify a bit, I see inline disposition condition at multiple places

Reply:
Refactored. Please review.